backend/services/immediate_cleanup.py

```
@@ -129,64 +60,29 @@ def get_cleanup_report() -> Dict[str, Any]:
-            'error': str(e),
-            'timestamp': datetime.now().isoformat()
-        }
-    
-    finally:
-        db.close()
+    # Since we're not using archive tables anymore, return empty stats
+    return {
+        'archive_stats': {},
+        'recommendation': "No cleanup needed - archive tables not in use",
+        'cleanup_available': False,
+        'timestamp': datetime.now().isoformat()
```

Ruff (0.14.1)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)